aws/credentials.go

```diff
@@ -90,6 +90,9 @@ type Credentials struct {
 	// The time the credentials will expire at. Should be ignored if CanExpire
 	// is false.
 	Expires time.Time
+
+	// AWS Account ID resolved from identity and used for optional endpoint2.0 routing
```

Comment:
We don't need to project what it's used for here:

```suggestion
	// The ID of the account for the credentials.
```